examples/apps/ai-collab/src/components/TaskGroup.tsx

```diff
@@ -48,6 +52,63 @@ export function TaskGroup(props: {
 	const [isDescriptionEditing, setIsDescriptionEditing] = useState(false);
 	const [isDiffModalOpen, setIsDiffModalOpen] = useState(false);
 
+	/**
+	 * Undo / Redo stacks of {@link Revertible} objects maintained for local branches.
+	 */
+	const [undoStack, setUndoStack] = useState<Revertible[]>([]);
+	const [redoStack, setRedoStack] = useState<Revertible[]>([]);
+
+	useEffect(() => {
+		function onDispose(disposed: Revertible): void {
```

Comment:
If I'm understanding the code in SharedTree correctly, this is going to get called when _the revertible_ is disposed, correct? If so, I think naming it `onRevertibleDisposed` like I see in SharedTree would be a good idea to make this code easier to understand without knowing ST.

Reply:
Yes.

It is used as the callback in [getRevertible](https://github.com/microsoft/FluidFramework/blob/af27b58c9a5a2e3bf3aa25779392c6a39d0963d6/packages/dds/tree/src/shared-tree/treeCheckout.ts#L447) of `TreeCheckout` class when `dispose()` is called on the `Revertible` instance.

Reply:
And I've changed the name to `onRevertibleDisposed()`.